<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSettings</class>
 <widget class="QDialog" name="DialogSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="tab">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab1">
        <attribute name="title">
         <string>Global Settings</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Close to Systray</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="cbCloseToSysTray">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QTimeEdit" name="idleTimeOut">
           <property name="displayFormat">
            <string>h:mm:ss</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Log Window Order By</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="logOrderBy"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Check Update every</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="cboUpdate"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Ilde timeout (hh:mm)</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Track window transparency</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSlider" name="transparency">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Calendars</string>
        </attribute>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Calendars</string>
         </property>
        </widget>
        <widget class="QListWidget" name="calendarList">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>231</width>
           <height>151</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="addCalendarButton">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>40</y>
           <width>85</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
        <widget class="QPushButton" name="removeCalendarButton">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>100</y>
           <width>85</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
        <widget class="QPushButton" name="editCalendarButton">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>70</y>
           <width>85</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Templates</string>
        </attribute>
        <widget class="QPushButton" name="removeTemplateButton">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>100</y>
           <width>85</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
        <widget class="QPushButton" name="addTemplateButton">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>40</y>
           <width>85</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
        <widget class="QPushButton" name="editTemplateButton">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>70</y>
           <width>85</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
        <widget class="QListWidget" name="templatesList">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>231</width>
           <height>151</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Templates</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
